specification/mysql/resource-manager/readme.md

```diff
@@ -161,7 +161,7 @@ csharp:
   azure-arm: true
   license-header: MICROSOFT_MIT_NO_VERSION
   namespace: Microsoft.Azure.Management.MySQL
-  output-folder: $(csharp-sdks-folder)/postgresql/Microsoft.Azure.Management.PostgreSQL/src/postgresql/Generated
+  output-folder: $(csharp-sdks-folder)/postgresql/Microsoft.Azure.Management.MySQL/src/postgresql/Generated
```

Comment:
Path should be 
output-folder: $(csharp-sdks-folder)/**mysql**/Microsoft.Azure.Management.MySQL/src/postgresql/Generated

Reply:
updated the path folder.